dashboard/app/controllers/javabuilder_sessions_controller.rb

```diff
@@ -13,40 +13,105 @@ class JavabuilderSessionsController < ApplicationController
 
   # GET /javabuilder/access_token
   def get_access_token
-    teacher_list = get_teacher_list.join(',')
+    begin
+      require_default_params
+      params.require(:channelId)
+    rescue ActionController::ParameterMissing
+      return render status: :bad_request, json: {}
+    end
     channel_id = params[:channelId]
+
+    begin
```

Comment:
`storage_id` and `storage_app_id` I believe can all be deleted, but I think we should do that when we fully remove the `useDashboardSources` flag from here, or as a separate PR.